pkg/operator/bootstrapteardown/teardown.go

```diff
@@ -20,13 +24,12 @@ import (
 )
 
 func TearDownBootstrap(config *rest.Config,
-	clusterMemberShipController *clustermembercontroller.ClusterMemberController, operatorClient v1helpers.OperatorClient) error {
+	clusterMemberShipController *clustermembercontroller.ClusterMemberController, etcdClient etcdv1.EtcdInterface) error {
 	failing := configv1.ClusterStatusConditionType("Failing")
 	var lastError string
-	var err error
 
 	cc := configclient.NewForConfigOrDie(config)
-	_, err = clientwatch.UntilWithSync(
+	_, _ = clientwatch.UntilWithSync(
```

Comment:
why dont we care about err?

Reply:
It was my understanding that because we never return error from the inner function we dont care about the error returned here.

Basically we retry until we don't succeed and ignore the error? do I need to change it? is it reasonable?

Reply:
if there is some unknown issue I think it would be good to know could we print the error if klog level 4?

Reply:
sure